go.mod

```diff
@@ -2,6 +2,9 @@ module github.com/grafana/grafana-plugin-sdk-go
 
 go 1.12
 
+// github.com/bombsimon/wsl/issues/60
+replace github.com/bombsimon/wsl => github.com/bombsimon/wsl v1.2.5
```

Comment:
Do we need this if we remove `-u` from go install?

Reply:
Nope we don't, will make that change before merging.